src/components/UpgradeBtn/UpgradeBtn.js

```diff
@@ -21,7 +21,12 @@ const Trigger = ({ className, children = 'Upgrade', ...props }) => (
 
 // NOTE(vanguard): redux passes "dispatch" prop to the component.
 // We should capture it in order to not assign it as a invalid dom attribute
-const UpgradeBtn = ({ isLoggedIn, dispatch, ...props }) => {
+const UpgradeBtn = ({
+  isLoggedIn,
+  title = 'Plan upgrade',
```

Comment:
@DmitriVanGuard . hello! :+1: 
For example, custom UpgradeBtn as 
![image](https://user-images.githubusercontent.com/14061779/63457403-a623ac80-c459-11e9-81b3-fd8df6a7e292.png)

Reply:
This `UpgradeBtn` (Get more data) has the same title - `Plan upgrade`. It was never changed since `title` was exposed as a prop. This component will only be used in the context of **plan upgrade**, therefore title of the opened dialog should be `"Plan dialog"`. Making some data as the prop (without a reason) negatively affect on the application size/performance (in the long run)